bp-esa-frontend/src/components/filterSearchForm.js

```diff
@@ -1,49 +1,61 @@
 import React, { Component } from 'react';
 import { Row, Col} from 'reactstrap';
 
-class FilterSearchForm extends Component {
-    render() {
-        return(
-            <div>
-                <form>
-                    <Row>
-                        <Col sm={3}>
-                        <fieldset className="form-group">
-                            <legend>Search Filter</legend>
-                            {
-                                 this.props.filterCriteria.map(criteria => (
-                                    <div className="form-check">
-                                    <label className="form-check-label">
-                                        <input
-                                        type="radio"
-                                        className="form-check-input"
-                                        name="filter"
-                                        id={criteria.name}
-                                        value={criteria.name}
-                                        />
-                                        {criteria.display}
-                                    </label>
-                                    </div>
-                                 ))
-                            }
-                        </fieldset>
-                        </Col>
-                        <Col sm={9}>
-                        <div className="form-group">
-                            <input
-                            type="search"
-                            className="form-control"
-                            placeholder="&#xF002; Search"
-                            id="search"
-                            name="search"
-                            />
-                        </div>
-                        </Col>
-                    </Row>
-                </form>
-            </div>
-        );
-    }
+const FilterSearchForm = (props) => {
```

Comment:
I think `Filters` would be a better name for this class. Ensure that it only deals with the filter criteria by moving the search bar and the status drop down to components of their own. Then place all of these related components in a folder. 

Also, having the search bar as a different component helps with reusability.